mockup/patterns/tinymce/less/pattern.tinymce.less

```diff
@@ -1,6 +1,6 @@
-@import (less) "@{bowerPath}/tinymce-builded/js/tinymce/skins/lightgray/skin.less";
-@import (less) "@{bowerPath}/tinymce-builded/js/tinymce/skins/lightgray/Content.Objects.less";
-@import (inline) "@{bowerPath}/tinymce-builded/js/tinymce/plugins/visualblocks/css/visualblocks.css";
+@import (inline) "@{bowerPath}tinymce-builded/js/tinymce/skins/lightgray/skin.min.css";
+@import (inline) "@{bowerPath}tinymce-builded/js/tinymce/skins/lightgray/content.min.css";
```

Comment:
@frapell Would you rememer any specific reason for using ``(inline)`` instead of ``(css)`` here? I found out webpack builds failing, because ``(inline)`` makes less-leader import context to be the folder of pattern and webpack cannot find images and fonts below the skin folder. I wonder how normal Plone works with that.

Reply:
@datakurre Honestly not... if you change it to ```(css)``` works? If so, I would say we can change it...